### DISCUSSION TITLE
Refresh DB Schemas Support

### DISCUSSION THREAD

**Discussion by learnasteve** [RamiAwar/dataline#320] [Q&A] [answered]
How can I update what dataline can see? Tried restarting docker. Had remove the delete the db connection in dataline (why does this also delete my chat history?!) and recreate it. Now it can see the new tables.

**Comment by RamiAwar** [comment, marked as answer]
UPDATE: We've added a refresh button, will be released in the next release! @learnasteve (sometime before Monday, still pending more changes)

Ahaa we were waiting for this issue 😄 You beat our development speed!

This'll be included in the next release - we wanted to ship the include/exclude asap and have people use it already. We didn't expect you to already use it + add new tables, impressive that you found this edge case.

Next release, we'll have auto-schema refresh + manual refresh button included.
We will still be auto-excluding new tables by default.

**Reply by learnasteve** [reply]
Amazing team! Well done. Love your work. :D
